src/main/java/org/openstreetmap/atlas/geography/atlas/complete/CompleteLine.java

```diff
@@ -257,6 +262,32 @@ public void setTags(final Map<String, String> tags)
         this.tags = tags != null ? new HashMap<>(tags) : null;
     }
 
+    @Override
+    public JsonObject toJson()
+    {
+        final JsonObject lineObject = new JsonObject();
+        lineObject.addProperty("identifier", this.identifier);
+        lineObject.addProperty("type", ItemType.LINE.toString());
+        lineObject.addProperty("geometry", this.polyLine.toString());
+
+        final JsonObject tagsObject = new JsonObject();
+        for (final String tagKey : new TreeSet<>(this.tags.keySet()))
```

Comment:
Is there a way to add the code for tags in a common method in a parent class? To avoid duplication?